interface/patient_file/summary/list_amendments.php

```diff
@@ -59,56 +50,65 @@ function checkUncheck(option) {
-    $resultSet = sqlStatement($query, array($pid));
-if (sqlNumRows($resultSet)) { ?>
-            <table class="w-100" cellspacing="0" cellpadding="0">
-                <tr>
-                    <td><a href="javascript:checkForAmendments();" class="btn btn-primary"><span><?php echo xlt("Print Amendments"); ?></span></a></td>
-                    <td align="right">
-                        <a href="#" class="small" onClick="checkUncheck(1);"><span><?php echo xlt('Check All');?></span></a> |
-                        <a href="#" class="small" onClick="checkUncheck(0);"><span><?php echo xlt('Clear All');?></span></a>
-                    </td>
-                </tr>
-            </table>
-        <div id="patient_stats">
-            <br />
-        <table border='0' cellpadding='0' cellspacing='0' style="margin-bottom:1em;">
-
-        <tr class='head'>
-            <th style="width:5%"></th>
-            <th style="width:15%" align="left"><?php echo  xlt('Requested Date'); ?></th>
-            <th style="width:40%" align="left"><?php echo  xlt('Request Description'); ?></th>
-            <th style="width:25%" align="left"><?php echo  xlt('Requested By'); ?></th>
-            <th style="width:15%" align="left"><?php echo  xlt('Request Status'); ?></th>
-        </tr>
-
-        <?php while ($row = sqlFetchArray($resultSet)) {
-            $amendmentLink = "<a href='add_edit_amendments.php?id=" . attr_url($row['amendment_id']) . "'>" . text(oeFormatShortDate($row['amendment_date'])) . "</a>";
-            ?>
-            <tr class="amendmentrow" id="<?php echo attr($row['amendment_id']); ?>">
-                <td><input id="check_list[]" name="check_list[]" type="checkbox" value="<?php echo attr($row['amendment_id']); ?>"></td>
-                <td class=text><?php echo $amendmentLink; ?> </td>
-                <td class=text><?php echo text($row['amendment_desc']); ?> </td>
-                <td class=text><?php echo generate_display_field(array('data_type' => '1','list_id' => 'amendment_from'), $row['amendment_by']); ?> </td>
-                <td class=text><?php echo generate_display_field(array('data_type' => '1','list_id' => 'amendment_status'), $row['amendment_status']); ?> </td>
-            </tr>
-        <?php } ?>
-        </table>
+<body>
+    <div class="container">
+        <div class="row">
+            <div class="col-12">
+                <h2><?php echo xlt('List'); ?></h2>
+            </div>
+            <div class="col-12">
+                <form action="list_amendments.php" name="list_amendments" id="list_amendments" method="post" onsubmit='return top.restoreSession()'>
+                <?php
+                $query = "SELECT * FROM amendments WHERE pid = ? ORDER BY amendment_date DESC";
+                $resultSet = sqlStatement($query, array($pid));
+                if (sqlNumRows($resultSet)) { ?>
+                    <table class="table w-100">
```

Comment:
We don't need `w-100` when the table class already sets it to 100%. Also add table-responsive here as well.